report/Introduction/ProjectProposal_eng.tex

```diff
@@ -0,0 +1,58 @@
+\section{Project Proposal}
+\subsection{Purpose}
+In agile processes there is a great focus on that requirements is not something that you just make, but something that you discover and improve throughout the project.
```

Comment:
```suggestion
In agile processes there are a great focus on that requirements is not something that you just make, but something that you discover and improve throughout the project.
```

Reply:
Jeg er altså ret sikker på at det skal være is